op-guide/ansible-deployment.md

```diff
@@ -318,6 +318,8 @@ To deploy TiDB using a normal user account, take the following steps:
 
 ## Test the cluster
 
+Note: Because TiDB is compatible with MySQL, you can use MySQL client to connect to TiDB directly. Before you start TiDB, make sure to install MySQL client first.
```

Comment:
Installing before starting TiDB is not necessary. I think the last sentence can be removed.

Reply:
If users don’t have MySQL client installed, how can they test their connection to TiDB? I believe TiDB doesn’t have a client yet...

Reply:
They need a MySQL client, yes! But they don't necessarily need one before installing TiDB. I think maybe `Because TiDB is compatible with MySQL, you must use MySQL client to connect to TiDB directly.` is appropriate?